content/docs/2.0/scalers/external-push.md

```diff
@@ -0,0 +1,49 @@
++++
+title = "External Push"
+layout = "scaler"
+availability = "v2.0+"
+maintainer = "Microsoft"
+description = "Scale applications based on an external push scaler."
+go_file = "external_scaler"
++++
+
+### Trigger Specification
+
+This specification describes the `external-push` trigger for an external push scaler.
+
+```yaml
+triggers:
+- type: external-push
+  metadata:
+    scalerAddress: external-scaler-service:8080
+    tlsCertFile: /path/to/tls/cert.pem # optional
+```
+
```

Comment:
I would add a link here maybe on how to write an external push scaler (concepts)?

Same goes for the external scaler.